packages/opentelemetry-cloud-monitoring-exporter/package.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Any reason to reformat to 4 spaces?

Reply:
This was a mistake, my text editing program acts out sometimes.